databases.go

```diff
@@ -277,6 +277,29 @@ func (c *Client) DatabaseCreateUser(ctx context.Context, app, addonID string, us
 	return res.DatabaseUser, nil
 }
 
+type DatabaseUpdateUserParam struct {
+	DatabaseID           string `json:"database_id"`
+	Password             string `json:"password,omitempty"`
+	PasswordConfirmation string `json:"password_confirmation,omitempty"`
+}
+
+type databaseUpdateUserPayload struct {
+	DatabaseUser DatabaseUpdateUserParam `json:"database_user"`
+}
+
+// DatabaseUpdateUser updates a user to the given database addon
+func (c *Client) DatabaseUpdateUser(ctx context.Context, app, addonID, username string, databaseUserParams DatabaseUpdateUserParam) (DatabaseUser, error) {
+	res := DatabaseUserResponse{}
+	payload := databaseUpdateUserPayload{
+		DatabaseUser: databaseUserParams,
+	}
+	err := c.DBAPI(app, addonID).SubresourceUpdate(ctx, "databases", addonID, "users", username, payload, &res)
```

Comment:
question: "databases" and "users" shouldn't be constants?

Reply:
I wonder, but IMO I prefer to see what is the route by seeing the string "databases".

But indeed it leaves room for potential manual errors.

However I think we should make this consistent with all other packages too.
It should be a dedicated issue I think.

Reply:
Here: https://github.com/Scalingo/go-scalingo/issues/375